File: Projects/Rock_Paper_Scissors/rps.java
import java.util.Random;
import java.util.Scanner;

// 0 is rock
// 1 is paper 
// 2 is scissors

public class rps {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        Random r = new Random();
        int computers_choice = r.nextInt(2);
        System.out.println("Type 0 for rock, 1 for paper and 2 for scissors ");
        int players_choice = sc.nextInt();

        System.out.print("Computer choice was: ");
        System.out.print(computers_choice);
        System.out.println();

        if(players_choice == computers_choice) {
            System.out.println("Draw!");
        }

        else if(players_choice == 0) {
            if(computers_choice == 1) {
                System.out.println("Computer won!");
            } else {
                System.out.println("You win!");
            }
        }

        else if(players_choice == 1) {
            if(computers_choice == 0) {
                System.out.println("You win!");
            } else {
                System.out.println("Computer won!");
            }
        }

        else if(players_choice == 2) {
            if(computers_choice == 0) {
                System.out.println("Computer won!");
            }
            else {
                System.out.println("You won!");
            }
        }
    }
}

File: Projects/Rock_Paper_Scissors/rps.py
import random

# 0 is rock
# 1 is paper 
# 2 is scissors
computers_choice = random.randint(0,2)
players_choice = int(input("Type 0 for rock, 1 for paper, or 2 for scissors "))
print("Computer played:", computers_choice)

if players_choice == computers_choice:
    print("Draw!")

elif players_choice == 0:
    if computers_choice == 1:
        print("Computer won!")
    else:
        print("You win!")

elif players_choice == 1:
    if computers_choice == 0:
        print("You win!")
    else:
        print("Computer won!")

elif players_choice == 2:
    if computers_choice == 0:
        print("Computer won!")
    else:
        print("You win!")
